spec/models/provider_user_spec.rb

```diff
@@ -78,6 +78,25 @@
     end
   end
 
+  describe '#can_view_safeguarding_information_for?' do
+    let(:provider_user) { create :provider_user, :with_provider }
+
+    it 'is false without the correct permission' do
+      expect(provider_user.can_view_safeguarding_information_for?(build_stubbed(:provider, id: 1))).to be false
```

Comment:
Would this be slightly better with:

```
      provider = provider_user.providers.first
      expect(provider_user.can_view_safeguarding_information_for?(provider)).to be false
```

Just to prove provider affiliation alone does not grant view rights.